### DISCUSSION TITLE
[table tree] 内包含 from 如何做 validate

### DISCUSSION THREAD

**Discussion by guoyong** [ant-design/ant-design#34403] [Q&A]
我有一个树形的table，每行有一些form元素，看上去类似这样
![image](https://user-images.githubusercontent.com/8455655/157596964-7d0abc71-d8ea-4109-9caf-8e6b6adbb1b7.png)

现在我有一个问题，当表单提交的时候收起的元素不会触发校验，getFieldsError 和 validateFields 均不会包含收起的元素。
![image](https://user-images.githubusercontent.com/8455655/157597764-24c27ec5-d51a-4477-a0a9-f0ce7e1928e5.png)

怎么才能正确的 处理被收起的元素内的formitem 校验？实际使用中我的数据层级可能比这个例子还要复杂的多

下面是一个demo
https://codesandbox.io/s/shu-xing-shu-ju-zhan-shi-antd-4-19-1-forked-0o7hte?file=/index.js:2789-2803

**Comment by linzixi31** [comment]
同问

**Reply by cuilingh00** [reply]
一年了，有解决方案吗？
